File: app/src/main/java/ir/ahfz/rentcar/io/network/model/ColorResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;

public class ColorResponse extends BaseResponse {

    private List<Color> colors = null;

    public List<Color> getColors() {
        return colors;
    }

    public void setColors(List<Color> colors) {
        this.colors = colors;
    }


    public static class Color {

        private int id;
        private String color;
        private String createdAt;
        private String updatedAt;

        public int getId() {
            return id;
        }

        public void setId(int id) {
            this.id = id;
        }

        public String getColor() {
            return color;
        }

        public void setColor(String color) {
            this.color = color;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

    }
}

File: app/src/main/java/ir/ahfz/rentcar/io/network/model/ColorResponse.kt
package ir.ahfz.rentcar.io.network.model

import ir.ahfz.rentcar.io.network.BaseResponse

/**
 * Data model for Branch response
 */
class ColorResponse(
    var colors: List<Color>? = null
) : BaseResponse() {
    /**
     * Data model for Branch response
     */
    data class Color(
        var id: Int = 0,
        var color: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null
    )
}
